source/plugins/go/input/lib/proxyutils.go

```diff
@@ -0,0 +1,91 @@
+package lib
+
+import (
+	"io/ioutil"
+	"os"
+	"strings"
+)
+
+var proxyCertPath = "/etc/ama-logs-secret/PROXYCERT.crt"
+
+type ProxyConfiguration struct {
+	User string
+	Pass string
+	Addr string
+	Port string
+}
+
+// func getProxyConfiguration() map[string]string {
+// 	amalogsproxySecretPath := "/etc/ama-logs-secret/PROXY"
+// 	if _, err := os.Stat(amalogsproxySecretPath); os.IsNotExist(err) {
+// 		return make(map[string]string)
+// 	}
+
+// 	proxyConfig, err := parseProxyConfiguration(amalogsproxySecretPath)
+// 	if err != nil {
+// 		// todo: handle error
+// 		return make(map[string]string)
+// 	}
+
+// 	// todo: log if empty
+// 	return proxyConfig
+// }
+
+func getProxyEndpoint() string {
```

Comment:
Did we validate all the HTTP proxy scenarios for both with AKS & Arc K8s clusters?